css/main-styling.css

```diff
@@ -0,0 +1,89 @@
+body {
+  font-family: 'Roboto Slab', serif;
+  font-size: 18px;
+  line-height: 1.5;
+  cursor: default;
+  font-color: black;
+}
+
+.wrapper {
+  max-width: 1280px;
+  margin: auto;
+}
+
+h1 {
+  font-size: 2em;
+  line-height: 1;
+}
+
+h3 {
+  font-size: 1.25em;
+  font-weight: bold;
+}
+
+p {
+  margin: 0 0 1.5em 0;
+  font-size: 14px;
```

Comment:
If you use `em` in one place for font-size, you shouldn't use `px` in another. Try using one font unit throughout your code.